pkgs/servers/x11/xorg/default.nix

```diff
@@ -2691,13 +2691,23 @@ lib.makeScope newScope (self: with self; {
     meta.platforms = stdenv.lib.platforms.unix;
   }) {};
 
-  xorgserver = callPackage ({ stdenv, pkgconfig, fetchurl, xorgproto, openssl, libX11, libXau, libXaw, libxcb, xcbutil, xcbutilwm, xcbutilimage, xcbutilkeysyms, xcbutilrenderutil, libXdmcp, libXfixes, libxkbfile, libXmu, libXpm, libXrender, libXres, libXt }: stdenv.mkDerivation {
```

Comment:
This file is generated, so you need to move your changes into `overrides.nix` in the same directory.

Reply:
Thanks for pointing that out :+1: